content/docs/15-compliance.md

```diff
@@ -36,6 +36,9 @@ Spectro Cloud is certified against SOC2 Type II, compliance with the AICPA’s (
 
 ## FIPS 140-2
 
+ 
+![FIPS-Compliance](compliance_FIPS.png "#width=180px")
+
 Spectro Cloud is certified against FIPS 140-2 with [Certificate number 4349](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4349) in compliance with the Cryptographic Module Validation Program (CMVP).  
 
 The Spectro Cloud Cryptographic Module is a general-purpose cryptographic library incorporated into the Kubernetes Management Platform and Kubernetes distributions for the protection of sensitive information.  
```

Comment:
Resma, at the end of this section, we should link to the new page you are creating.